fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/MetricLoader.java

```diff
@@ -18,7 +21,12 @@
      * @deprecated in favor of loadMetricDictionary(MetricDictionary, DimensionDictionary)
      */
     @Deprecated
-    void loadMetricDictionary(MetricDictionary metricDictionary);
+    default void loadMetricDictionary(MetricDictionary metricDictionary) {
+        String message = "The method loadMetricDictionary(MetricDictionary, DimensionDictionary) must be implemented " +
+                "for MetricLoader";
+        LoggerFactory.getLogger(DimensionToNameSerializer.class).error(message);
```

Comment:
@michael-mclawhorn so...? you are saying user don't have to implement `loadMetricDictionary` method at all in the `MetricLoader` and the loader is still perfectly healthy?